packages/commercetools/composables/src/index.ts

```diff
@@ -17,5 +17,6 @@ export { useUserBilling, useUserBillingFactoryParams } from './useUserBilling';
 export { useUserOrder, useUserOrderFactoryParams } from './useUserOrder';
 export { useUserShipping, useUserShippingFactoryParams } from './useUserShipping';
 export { useWishlist, useWishlistFactoryParams } from './useWishlist';
+export { default as useForgotPassword } from './useForgotPassword';
```

Comment:
I see the team began exporting FactoryParams from composables. I've just sent a question in our internal devs channel to understand why. It is highly probable, you will need to also export factoryParams for useForgotPassword

Reply:
yeah, please export factory params too